### PR TITLE
Cleanup unused code from NuGet.CommandLine

### PR DESCRIPTION
**Pull request by skofman1**
Removed some unused code from NuGet.CommandLine. Mostly those are private methods that are not used. Verified using UTs.

@emgarten, @yishaigalatzer

### CODE REVIEW

**Comment by alpaix** [comment]
looks good :shipit:

File: src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs [hunk 1]
```diff
@@ -92,7 +92,6 @@ internal void CalculateEffectivePackageSaveMode()
             {
                 try
                 {
-                    var xDocument = XDocument.Load(projectConfigFilePath);
                     var reader = new PackagesConfigReader(XDocument.Load(projectConfigFilePath));
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs]: use the variable don't delete the line

Comment (skofman1) [inline src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs]: Fixed

**Comment by skofman1** [comment]
merged

**Pull request closed without merging**